github_api_helper.js

```diff
@@ -118,22 +118,51 @@ function getConclusion (annotations) {
  * @param {import('probot').Context} context Probot context
  * @param {Number} runID check run identifier
  * @param {Object} output merged scan output
+ * @param {Number} annotationsPerPage Optional: number of annotations to send with one API call.
  * @returns {Promise<any>} GitHub response
  * See: https://developer.github.com/v3/checks/runs/#update-a-check-run
  */
-function sendResults (context, runID, output) {
+function sendResults (context, runID, output, annotationsPerPage) {
   const { owner, repo } = context.repo()
-  const completedAt = new Date().toISOString()
+  let numAnnotationsLeftToSend = output.annotations.length
+  const MAX_ANNOTATIONS_CALL = 50
```

Comment:
config.js also defines a 50 const.  Can we use only one definition of 50?

Reply:
Yes, but I thought that one day we can decide to change the const numAnnotationsPerUpdate in the config.js and then we need to have the const MAX_ANNOTAIONS_CALL for the check.

Reply:
Ah I see.